extensions/undertow/deployment/pom.xml

```diff
@@ -39,6 +39,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-undertow-spi</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-security-test-utils</artifactId>
```

Comment:
Shouldn't this be scope test?

Reply:
It's managed in the bom

Reply:
I added it there too to be on the safe side if we ever remove it.